src/toolbar-dropdown/index.js

```diff
@@ -63,15 +60,16 @@ const ToolbarDropdown = ( {
 						{ options.map( ( option ) => {
 							const isSelected =
 								option.value === selectedOption.value;
+							let itemClassNames = 'syntaxhighlighter-toolbar-dropdown__option';
+							if ( isSelected ) {
+								itemClassNames += ' is-selected';
+							}
```

Comment:
Just curious why `classnames` needed to be removed? Personally I think its cleaner to use that instead of manual string building, from a readability perspective.

Reply:
It was in order to clean up dependencies for things marginally used, seems a bit overkill to use a 3rd party npm package to do a string concatenation ... once 😅

Reply:
> once

Good point!